dist/react-json-schema.js

```diff
@@ -1,14 +1,591 @@
-'use strict';
```

Comment:
Looks like the distribution file is packaging in dependencies. I believe the webpack file should be updated to mark these as external.